index.bs

```diff
@@ -75,8 +78,8 @@ Note: This property should provide guarantees that are, at least, as strong as <
 
 <pre class=propdef>
 Name: scroll-boundary-behavior-x, scroll-boundary-behavior-y
-Value: ''contain'' | ''none'' | ''auto''
-Initial: ''auto''
+Value: contain | none | auto
```

Comment:
Thank you for fixing this. I recall trying several permutations and giving up hoping someone more bikeshed knowledge would clean it up.